enterprise/reporting/event_stats_test.go

```diff
@@ -181,6 +197,17 @@ func TestEventStatsReporter(t *testing.T) {
 		"terminal":        "false",
 		"status":          "non-terminal",
 	}).LastValue(), float64(100))
+	require.Equal(t, statsStore.Get(EventsProcessedMetricName, map[string]string{
```

Comment:
[optional]: 
Since we have added `statsStore.GetAll()`, we can add one more assertion here for:
```GO
	require.Len(t, statsStore.GetAll(), 5)
```